addons/bitcoin/src/constants.rs

```diff
@@ -0,0 +1 @@
+pub const NAMESPACE: &str = "btc";
```

Comment:
I think the namespace should be `bitcoin`, `btc` is a ticker.

Reply:
Bitcoin is _soooo_ much to type, and to build a script you have to type it a lot. This is just for a p2pkh:
```hcl
action "my_script" "btc::encode" {
    opcodes = [
        bitcoin::op_dup(),
        bitcoin::op_hash160(),
        bitcoin::op_pushdata(20, "55ae51684c43435da751ac8d2173b2652eb64105"),
        bitcoin::op_equalverify(),
        bitcoin::op_checksig()
    ]
}
```

I'm a bit torn on this - agree that btc is a ticker so maybe this is technically incorrect, but btc also has grown to mean more than just a ticker

Reply:
Verbosity, point taken, but this would be true for any chain - `sol` for `solana`, `stx` for `stacks`, etc.
It feels like we'd be setting a precedent.

Reply:
Agreed, we'd be setting a precedence, one that I like 🙂 I'd be down for `sol`/`stx`/`btc`

Reply:
I've changed the namespace to `bitcoin`